amundsen_application/static/js/components/Tags/TagsList/index.tsx

```diff
@@ -0,0 +1,147 @@
+// Copyright Contributors to the Amundsen project.
+// SPDX-License-Identifier: Apache-2.0
+
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+
+import ShimmeringTagListLoader from 'components/common/ShimmeringTagListLoader';
+import TagInfo from 'components/Tags/TagInfo';
+import { Tag } from 'interfaces';
+
+import {
+  POPULAR_TAGS_TITLE,
+  CURATED_TAGS_TITLE,
+  BROWSE_MORE_TAGS_TEXT,
+  OTHER_TAGS_TITLE,
+  BROWSE_TAGS_TITLE,
+  BROWSE_PAGE_PATH,
+} from './constants';
+import './styles.scss';
+
+export type TagsListProps = StateFromProps & OwnProps;
+
+export interface StateFromProps {
+  curatedTags: Tag[];
+  popularTags: Tag[];
+  otherTags?: Tag[];
+  isLoading?: boolean;
+}
+
+interface OwnProps {
+  /* determine if we only want curated/popular tags with
+  'Browse more tags' link or the entire tags list */
+  shortTagsList?: boolean;
+}
+
+interface TagsListTitleProps {
+  titleText: string;
+}
+
+const TagsListTitle: React.FC<TagsListTitleProps> = ({
+  titleText,
+}: TagsListTitleProps) => (
+  <h1 className="title-1" id="browse-header">
+    {titleText}
+  </h1>
+);
+
+const TagsListLabel: React.FC<TagsListTitleProps> = ({
+  titleText,
+}: TagsListTitleProps) => (
+  <label className="section-label">
+    <span className="section-title title-2">{titleText}</span>
+  </label>
+);
+
+interface TagsListBlockProps {
+  tags: Tag[];
+}
+
+const TagsListBlock: React.FC<TagsListBlockProps> = ({
+  tags,
+}: TagsListBlockProps) => {
+  return (
+    <div id="tags-list" className="tags-list">
+      {tags.map((tag, index) => (
+        <TagInfo data={tag} compact={false} key={index} />
```

Comment:
Using the index as the key is not a good practice. If the tags are unique, we could use the name of them, something like: `key={`tagKey:${tag.name}`}`

Reply:
They should be unique but I don't know that we have any way of guaranteeing that someone won't add repeated tags on the custom config

Reply:
Still better than using the index I guess